ci/setup-gdk.ps1

```diff
@@ -1,129 +1,18 @@
-pushd "$($gdk_home)"
+# Expects gdk_home, which is not the GDK location in the engine
+param (
+    [string] $unreal_path = "$((Get-Item `"$($PSScriptRoot)`").parent.parent.FullName)\UnrealEngine",
+    [string] $msbuild_path = "$((Get-Item 'Env:programfiles(x86)').Value)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe"
+)
 
-    # Setup variables
-    $pinned_core_sdk_version = Get-Content -Path "$($gdk_home)\SpatialGDK\Extras\core-sdk.version" -Raw
-    $build_dir = "$($gdk_home)\SpatialGDK\Build"
-    $core_sdk_dir = "$($build_dir)\core_sdk"
-    $worker_sdk_dir = "$($gdk_home)\SpatialGDK\Source\SpatialGDK\Public\WorkerSDK"
-    $worker_sdk_dir_old = "$($gdk_home)\SpatialGDK\Source\Public\WorkerSdk"
-    $binaries_dir = "$($gdk_home)\SpatialGDK\Binaries\ThirdParty\Improbable"
-    $msbuild_exe = "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\MSBuild.exe"
+$gdk_in_engine = "$unreal_path\Engine\Plugins\UnrealGDK"
 
-    Write-Log "Creating folders.."
-    New-Item -Path "$($worker_sdk_dir)" -ItemType Directory -Force
-    New-Item -Path "$($core_sdk_dir)\schema" -ItemType Directory -Force
-    New-Item -Path "$($core_sdk_dir)\tools" -ItemType Directory -Force
-    New-Item -Path "$($core_sdk_dir)\worker_sdk" -ItemType Directory -Force
-    New-Item -Path "$($binaries_dir)" -ItemType Directory -Force
-    New-Item -Path "$($binaries_dir)\Programs" -ItemType Directory -Force
+# link the gdk into its expected place in the engine
+New-Item -ItemType Junction -Path "$gdk_in_engine" -Target "$gdk_home" -ErrorAction SilentlyContinue
```

Comment:
Are we fine continuing if this step fails?

Reply:
The silent continue is there to ignore any errors thrown when attempting to copy over files that already exist at the target location. Since we're merging folders, in local testing I've found that to be necessary. I think this should be fine since we'll fail later if the assembled GDK is somehow broken, otherwise an alternative would be to somehow overwrite existing files.

Reply:
So I have found that this and the other symlinks were slightly wrong, changed all and allowed errors to be thrown when creating them